tests/by-util/test_factor.rs

```diff
@@ -29,6 +29,8 @@ const NUM_TESTS: usize = 100;
 
 #[test]
 fn test_parallel() {
+    use sha1::{Sha1, Digest};
```

Comment:
please run rustfmt

Reply:
To expand on this, `cargo fmt` needs to be given the path to the tests explicitly to format them:
```shell
cargo fmt -- tests/**/*.rs
```

Reply:
Many thanks for the tip - I had not passed the correct parameters.